workflow/hpxml_inputs.json

```diff
@@ -1962,6 +1958,24 @@
     "hvac_control_heating_weekday_setpoint": "64, 64, 64, 64, 64, 64, 64, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 64, 64",
     "hvac_control_heating_weekend_setpoint": "64, 64, 64, 64, 64, 64, 64, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 64, 64"
   },
+  "sample_files/base-hvac-air-to-air-heat-pump-1-speed-lockout-temperatures-research-features.xml": {
+    "parent_hpxml": "sample_files/base-hvac-air-to-air-heat-pump-1-speed-lockout-temperatures.xml",
+    "simulation_control_timestep": 1,
+    "heat_pump_heating_capacity": 10000,
```

Comment:
I assume you are setting a very low heating capacity so that the backup heat triggers a lot? It ends up being a heat pump with 10k heating capacity and 36k cooling capacity, which doesn't reflect what you would possibly see in reality. Would it be better to set the compressor lockout temperature a bit above the heating design temperature? That should similarly cause large backup heating usage during cold periods.

Reply:
Yes it's set in this weird way to trigger back up heat more. The lockout temperature will help trigger backup coil and see the staging of backup coil, but the behavior to switch to backup coil when main coil is not able to maintain setpoint will be missing. What about keeping 36k heating and cooling but using a more extreme climate?

Reply:
Ah, good point. Yes, you could try using Duluth for the climate. You could also decrease the heat pump capacity a bit if you really need to, but 10k is quite small.